### PR TITLE
649 get avatar api

### PR DESCRIPTION
**Pull request by rpoletaev**

### CODE REVIEW

File: packages/api/api_test.go [hunk 1]
```diff
@@ -279,26 +279,46 @@ func cutErr(err error) string {
...
-			go func(counter int) {
-				defer wg.Done()
-
-				form := url.Values{`Amount`: {strconv.FormatInt(int64(i+1), 10)}, `Recipient`: {`1028-0432-0934-8475-1098`}} //-8166311980224800518
-				if err := postTx(`MoneyTransfer`, &form); err != nil {
-					fmt.Println(err)
-				}
-			}(j)
-		}
+	// err := model.GormInit("localhost", 5432, "postgres", "postgres", "apla")
```
Comment (potehinre) [inline packages/api/api_test.go]: commented code prohibited, even in tests

File: packages/model/binary.go [hunk 2]
```diff
@@ -28,6 +28,14 @@ func (b *Binary) Get(appID, memberID int64, name string) (bool, error) {
...
 }
 
+// Link returns link to binary data
 func (b *Binary) Link() string {
 	return fmt.Sprintf(`/data/%s/%d/%s/%s`, b.TableName(), b.ID, "data", b.Hash)
 }
+
+// GetByID is retrieving model from db by id
+func (b *Binary) GetByID(id int64) (bool, error) {
+	DBConn.LogMode(true)
```
Comment (potehinre) [inline packages/model/binary.go]: WTF?

File: packages/api/member.go [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+)
+
+func getAvatar(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.Entry) error {
+	parMember := data.params["member"].(string)
+	memberID := converter.StrToInt64(parMember)
+	member := &model.Member{}
+	member.SetTablePrefix(converter.Int64ToStr(data.ecosystemId))
+	found, err := member.Get(memberID)
+	if err != nil {
+		log.WithFields(log.Fields{"type": consts.DBError, "error": err}).
```
Comment (potehinre) [inline packages/api/member.go]: do not use Errorf, if you want log some ids, use WithFields

File: packages/api/member.go [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+	if member.ImageID == nil {
+		return errorAPI(w, "E_SERVER", http.StatusNotFound)
+	}
+
+	bin := &model.Binary{}
+	bin.SetTablePrefix(converter.Int64ToStr(data.ecosystemId))
+	found, err = bin.GetByID(*member.ImageID)
+	if err != nil {
+		log.WithFields(log.Fields{"type": consts.DBError, "error": err}).Errorf("on getting binary by id %d", *member.ImageID)
```
Comment (potehinre) [inline packages/api/member.go]: same stuff

**Review by potehinre — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Review by potehinre — APPROVED** [review]

**Pull request merged**